index.html

```diff
@@ -136,12 +136,170 @@ <h2>
         sensing, depth sensing, video analysis, gesture recognition, and so on.
       </p>
     </section>
+    <section>
+      <h2>
+        Model
+      </h2>
+      <p>
+        A <dfn>gamepad</dfn> is a collection of input controls (buttons,
+        triggers, joysticks, thumbsticks, touch surfaces) and output controls
+        (haptic actuators). A [=gamepad=] is <dfn>available</dfn> if the [=user
+        agent=] can read the current state of its input controls. A [=gamepad=]
+        that is not [=available=] is <dfn>unavailable</dfn>. The collection of
+        input and output controls for a [=gamepad=] cannot change while the
+        [=gamepad=] is [=available=]. Each input control has one or more
+        <dfn>input values</dfn> that may update over time.
+      </p>
+      <p>
+        The [=user agent=] is responsible for enumerating [=available=]
+        [=gamepads=], detecting when [=gamepads=] become [=available=] or
+        [=unavailable=], and detecting when input controls have updated input
+        values.
+      </p>
+      <p>
+        A [=gamepad=] has a <dfn>gamepad identifier string</dfn>, a
+        human-readable string that identifies the brand or style of
+        [=gamepad=]. The content is decided by the [=user agent=].
+      </p>
+      <p>
+        A [=gamepad=] may be a physical device. If so, it has an <dfn>input
+        control layout</dfn> that describes the position, orientation and type
+        of each input control on the physical device. The [=user agent=] is
+        responsible for recognizing when an [=input control layout=]
+        <dfn data-lt="corresponds with">corresponds with a standard
+        layout</dfn>, meaning the relative positions and orientations of its
+        input controls are close enough to the standard layout that a user can
```

Comment:
Am I misreading, or "so" is missing here?
```suggestion
        input controls are close enough to the standard layout so that a user can
```

Reply:
I think omitting "so" is correct but I'm not satisfied with this definition and I think it should be rewritten to be clearer.

Maybe split it into two sentences:

The user agent is responsible for recognizing when an input control layout corresponds with a standard layout, meaning gamepads with this layout can be used interchangeably with other gamepads with layouts that correspond with the same standard layout. The user agent SHOULD consider a layout to correspond with a standard layout if its input controls have approximately the same relative positions and orientations as input controls described in the standard layout.

Reply:
sgtm